### PR TITLE
Feature/eco 182 error handling

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

**Comment by dereuromark** [comment]
did you run the code sniffer?

**Comment by ghost** [comment]
No, I have done it now and fixed two issues. Thanks.

**Comment by dereuromark** [comment]
You can and probably should the code sniffer as require dev dependency as I did here: https://github.com/spryker-eco/computop/pull/2
It allows then to quickly run `composer cs-check/fix` on it :)
Also note that using .license file you can now also assert the MIT license handling automatically.

File: src/SprykerEco/Yves/FactFinder/Controller/SearchController.php [hunk 1]
```diff
@@ -35,6 +35,10 @@ public function indexAction(Request $request)
             ->getFactFinderClient()
             ->search($factFinderSearchRequestTransfer);
 
+        if (empty($ffSearchResponseTransfer->getResult())) {
```
Comment (dereuromark) [inline src/SprykerEco/Yves/FactFinder/Controller/SearchController.php]: Please dont use empty() with methods. Use ! instead.

File: src/SprykerEco/Yves/FactFinder/Controller/SuggestionsController.php [hunk 2]
```diff
@@ -33,6 +33,10 @@ public function indexAction(Request $request)
             ->getFactFinderClient()
             ->getSuggestions($factFinderSuggestRequestTransfer);
 
+        if (empty($response->getSuggestions())) {
```
Comment (dereuromark) [inline src/SprykerEco/Yves/FactFinder/Controller/SuggestionsController.php]: Same

File: src/SprykerEco/Yves/FactFinder/Controller/SearchController.php (hunk 1 above)
Comment (ghost) [reply src/SprykerEco/Yves/FactFinder/Controller/SearchController.php]: Ok, no problems)

**Pull request merged**